server/routerlicious/packages/routerlicious-base/src/alfred/routes/api/api.ts

```diff
@@ -129,6 +135,20 @@ export function create(
 		},
 	);
 
+	router.post("/broadcast-signal", (req, res) => {
+		try {
+			const tenantId = req.body?.tenantId;
+			const documentId = req.body?.documentId;
+			const signalContent = req.body?.singalContent;
+			const signalRoom: IRoom = { tenantId, documentId };
+			const payload: IBroadcastSignalEventPayload = { signalRoom, signalContent };
+			collaborationSessionEventEmitter.emit("broadcast-signal", payload);
+			res.status(200).send("Triggering debug signal from tinylicious");
```

Comment:
Sorry if I missed any context about this, but is this something that was needed only for testing purposes? In other words, the message says `Triggering debug signal from tinylicious` - is this route something we want to be available always in routerlicious?